src/goGetPackage.ts

```diff
@@ -23,24 +22,17 @@ export function goGetPackage() {
 		return;
 	}
 
-	const goRuntimePath = getBinPath('go');
-	if (!goRuntimePath) {
-		return vscode.window.showErrorMessage(`Failed to run "go get" to get package as the "go" binary cannot be found in either GOROOT(${process.env['GOROOT']}) or PATH(${envPath})`);
-	}
-
 	const env = Object.assign({}, process.env, { GOPATH: getCurrentGoPath() });
+	const result = await goGet(importPath, ['-v'], { env });
```

Comment:
`result` can be `null` when path to go binary is not found.

Reply:
Added error handling for this.